charts/sonarqube-dce/README.md

```diff
@@ -221,17 +221,17 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `searchNodes.readinessProbe.periodSeconds`                | ReadinessProbe period between checking Search Node                                    | `30`                                                                   |
-| `searchNodes.readinessProbe.failureThreshold`             | ReadinessProbe thresold for marking as failed                                         | `6`                                                                    |
+| `searchNodes.readinessProbe.failureThreshold`             | ReadinessProbe threshold for marking as failed                                         | `6`                                                                    |
 | `searchNodes.readinessProbe.timeoutSeconds`               | ReadinessProbe timeout delay                                                          | `1`                                                                    |
-| `searchNodes.livenessProbe.initialDelaySeconds`           | LivenessProbe initial delay for Search Node checking                                  | `60`                                                                   |
+| `searchNodes.livenessProbe.initialDelaySeconds`           | LivenessProbe initial delay for Search Node checking                                  | `0`                                                                   |
 | `searchNodes.livenessProbe.periodSeconds`                 | LivenessProbe period between checking Search Node                                     | `30`                                                                   |
-| `searchNodes.livenessProbe.failureThreshold`              | LivenessProbe thresold for marking as dead                                            | `6`                                                                    |
+| `searchNodes.livenessProbe.failureThreshold`              | LivenessProbe threshold for marking as dead                                            | `6`                                                                    |
 | `searchNodes.livenessProbe.timeoutSeconds`                | LivenessProbe timeout delay                                                           | `1`                                                                    |
-| `searchNodes.startupProbe.initialDelaySeconds`            | StartupProbe initial delay for Search Node checking                                   | `30`                                                                   |
+| `searchNodes.startupProbe.initialDelaySeconds`            | StartupProbe initial delay for Search Node checking                                   | `20`                                                                   |
 | `searchNodes.startupProbe.periodSeconds`                  | StartupProbe period between checking Search Node                                      | `10`                                                                   |
```

Comment:
searchNodes.startupProbe.periodSeconds looks like is at 5 instead of 10

Reply:
Hey @jCOTINEAU, it's actually at 10 in the value file..